### DISCUSSION TITLE
Images in different directories

### DISCUSSION THREAD

**Discussion by siiigrid** [mikel-brostrom/yolo_tracking#749] [Q&A]
Is there any way to execute it with all my images at once if they are in directories like this?:
.
└── images/
    ├── 1/
    │   ├── im1.png
    │   ├── im2.png
    │   └── im3.png
    ├── 2/
    │   ├── im4.png
    │   ├── im5.png
    │   └── im6.png
    └── 3/
        ├── im7.png
        ├── im8.png
        └── im9.png

**Comment by mikel-brostrom** [comment]
You can create a `.txt` file containing you stream sources

```
path/to/1/
path/to/2/
path/to/3/
```

finally you can run

python3 track.py --source streams.txt
